build.gradle

```diff
@@ -2,17 +2,19 @@
 
 buildscript {
     repositories {
+        google()
```

Comment:
It's a shortcut for google's maven - I just wasn't able to download the support libraries from gradle without it - so just ignore it and do what works for you on Studio 2